docs/samples/Microsoft.ML.Samples/Program.cs

```diff
@@ -15,7 +15,7 @@ internal static void RunAll()
             {
                 var sample = type.GetMethod("Example", BindingFlags.Public | BindingFlags.Static | BindingFlags.FlattenHierarchy);
 
-                if (sample != null)
+                if (sample != null && type.Name == "ResnetV2101TransferLearningTrainTestSplit")
                 {
```

Comment:
Did this get accidentally left behind?

Reply:
That's right, thank you!

---
In reply to: [337586213](https://github.com/dotnet/machinelearning/pull/4362#discussion_r337586213) [](ancestors = 337586213)